### PR TITLE
Fix some typos

### PR DESCRIPTION
**Pull request by memoryleak47**

### CODE REVIEW

File: lang/values.md [hunk 1]
```diff
@@ -321,9 +321,9 @@ impl PartialOrd for Value {
                 p1 <= p2,
             (Tuple(vals1), Tuple(vals2)) =>
                 vals1 <= vals2,
-            (Variant { idx: idx1, data: data1 }, Variant { idx: idx2, data: data2 }) =>
-                idx == idx1 && data1 <= data2,
-            (Union(chunks1), Union(chunks2)) => chunks1 <= chunks2,
+            (Value::Variant { idx: idx1, data: data1 }, Value::Variant { idx: idx2, data: data2 }) =>
```
Comment (RalfJung) [inline lang/values.md:324]: Just above we use the other `Value` variants so this doesn't seem right.

Can we do `use Value::*;` at the top of the function instead?

**Review by RalfJung — COMMENTED** [review]
Thanks! Looks good aside from one point.

File: lang/values.md (hunk 1 above)
Comment (RalfJung) [reply lang/values.md:324]: This should probably not be in this PR anyway, it conceptually seems to belong to your other PR.

Comment (memoryleak47) [reply lang/values.md]: Yep, this shouldn't be in this PR. good catch!

**Comment by memoryleak47** [comment]
I fixed the commit, and then rebased ontop of RalfJung/master.

**Comment by RalfJung** [comment]
Thanks! Please squash this into a single commit.

**Comment by memoryleak47** [comment]
done!

**Pull request merged**